### PR TITLE
Completed laboratory work No. 4

### PR DESCRIPTION
**Pull request by ViktorovVladimir**
Part 1
1. Write a program that prints odd numbers from 1 to 99 to the console. Each number is printed on a new line.
2. Write a program that prints numbers from 1 to 100 that are divisible by 3, 5, and both (that is, both 3 and 5). Example output:
	Divisible by 3: ….
	Divisible by 5: ….
	Divisible by 3 and 5: ….
3. Write a program to calculate the sum of two integers and return true if the sum is equal to the third integer. Example output:
	Enter first number: 5
	Enter second number: 10
	Enter third number: 15
	Result: true
4. Write a program that accepts three integers from the user and returns true if the second number is greater than the first number and the third number is greater than the second number. Example output:	
	Enter first number: 5
	Enter second number: 10
	Enter third number: 15
	Result: true
5. Write a program to check whether the number 3 appears as the first or last element of an array of integers. The length of the array must be greater than or equal to two. Example output:
	array = 3, -3, 7, 4, 5, 4, 3
	true
6. Write a program to check if an array contains the number 1 or 3.

Part 2
!Do not use ready-made methods for sorting, searching for an element, etc.
1. Write a program that checks whether the array is sorted in ascending order. If it is sorted in ascending order, “OK” will be displayed, if not, then the text “Please, try again” will be displayed.
2. Write a program that reads the length of an array from the keyboard (for example, the user enters the number 4), then the user enters 4 numbers and an array of 4 elements is printed on a new line. Example output:
	Array length: 4
	Numbers of array:
	5
	6
	7
	2
	Result: [5, 6, 7, 2]
3. Write a method that swaps the first and last element of an array. Example output:
	Array 1: [5, 6, 7, 2]
	Array 2: [2, 6, 7, 5]
4. An array of numbers is given. Find the first unique number in this array. 
For example, for the array [1, 2, 3, 1, 2, 4], this is the number 3.
5. Fill the array with random numbers and sort it. Use merge sort.
6. ***Read chapters 1-4 “Grocking Algorithms”

### CODE REVIEW

File: src/LabWork04/Part02/Task03/Main.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package LabWork04.Part02.Task03;
+
+import java.util.Arrays;
+
+public class Main {
+
+    public static boolean funcCompare(int[] mas) {
```
Comment (ArthurKharatian) [inline src/LabWork04/Part02/Task03/Main.java]: тут метод таки не является проверкой, он просто меняет местами

File: src/LabWork04/Part02/Task04/Main.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package LabWork04.Part02.Task04;
+
+import java.util.Arrays;
+
+public class Main {
+
+    public static int funcCompare(int[] mas) {
+        if (mas.length < 2) {
+            return -1;
```
Comment (ArthurKharatian) [inline src/LabWork04/Part02/Task04/Main.java:9]: а вот тут тип boolean, что нам делать с -1?

File: src/LabWork04/Part02/Task05/Main.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
+package LabWork04.Part02.Task05;
+
+import java.util.Arrays;
+
+public class Main {
+
+    public static boolean funcBubbleSort(int[] mas) {
```
Comment (ArthurKharatian) [inline src/LabWork04/Part02/Task05/Main.java:7]: тут тож какой смысл возвращать булеан?

Comment (ViktorovVladimir) [reply src/LabWork04/Part02/Task05/Main.java:7]: True - это состояние того, что процесс сортировки прошёл успешно;
False - возникли какие-то исключения;

**Pull request merged**